api/src/places/data/country-codes-full.json

```diff
@@ -0,0 +1,2652 @@
+[
+  {
+    "Code": "AF",
+    "Name": "Afghanistan",
+    "locales": [
+      {
+        "locale_code": "en-US",
+        "name": "Afghanistan"
+      },
+      {
+        "locale_code": "es-EC",
+        "name": "Afganist\u00c3\u00a1n"
```

Comment:
Did something strange happened here? What are these Unicode characters??

Reply:
I only did a rename from `country-codes.json` to `country-codes-full.json` so that the related test will take less time to complete. I believe those unicode characters are the accented characters or characters in other languages.